explainer.md

```diff
@@ -518,6 +518,46 @@ File API.
+
+### URL Behavior
+URLs without a trailing `/` are treated as if the final path segment had been removed for cookie read operations, including change monitoring. Paths for cookie read operations are resolved relative to the default read cookie path.
+
+### Secure Cookies
+This API only allows writing "Secure" cookies. This is intended to prevent unintentional leakage to unsecured connections on the same domain. Furthermore it disallows (to the extent permitted by the browser implementation) creation or modification of `Secure-`flagged cookies from unsecured web origins and enforces special rules for the [__Host-](https://tools.ietf.org/html/draft-ietf-httpbis-rfc6265bis#section-4.1.3.2) and [__Secure-](https://tools.ietf.org/html/draft-ietf-httpbis-rfc6265bis#section-4.1.3.1) cookie name prefixes.
+
+### Domain Defaults
+This API defaults cookies to "Domain"-less, which in conjunction with "Secure" provides origin-scoped cookie
+behavior in most modern browsers. When practical the `__Host-` cookie name prefix should be used with these cookies so that cooperating browsers origin-scope them.
+
+### Serializing Expiration Dates
+Serialization of expiration times for non-session cookies in a special cookie-specific format has proven cumbersome,
+so this API allows JavaScript Date objects and numeric timestamps (milliseconds since the beginning of the Unix epoch) to be used instead. The inconsistently-implemented Max-Age parameter is not exposed, although similar functionality is available for the specific case of expiring a cookie.
+
+### HTTP Cookie Header Serialization
+Cookies without U+003D (=) code points in their HTTP Cookie header serialization are treated as having an empty name, consistent with the majority of current browsers. Cookies with an empty name cannot be set using values containing U+003D (=) code points as this would result in ambiguous serializations in the majority of current browsers.
+
+### Interpreting Strings
+Internationalized cookie usage from scripts has to date been slow and browser-specific due to lack of interoperability because although several major browsers use UTF-8 interpretation for cookie data, historically Safari and browsers based on WinINet have not. This API mandates UTF-8 interpretation for cookies read or written by this API.
+
+### Monitoring Changes
+Use of cookie-change-driven scripts has been hampered by the absence of a power-efficient (non-polling) API for this. This API provides observers for efficient monitoring in document contexts and interest registration for efficient monitoring in service worker contexts.
+
+### Writing Cookies
+Scripts should not have to write and then read "test cookies" to determine whether script-initiated cookie write access is possible, nor should they have to correlate with cooperating server-side versions of the same write-then-read test to determine that script-initiated cookie read access is impossible despite cookies working at the HTTP level.
```

Comment:
"Scripts should not have to write and then read "test cookies" to determine whether script-initiated cookie write access is possible" -- I read this as providing some `canWriteCookie` API, which we haven't.

I'm guessing this entire section aims to say that we choose to fail explicitly on writing instead of silently like `document.cookie` does. I don't think this qualifies as an opinion, as this behavior is consistent with every modern storage API.